services/sleeper/metadata/metadata.yml

```diff
@@ -68,3 +79,10 @@ outputs:
       title: Random sleep interval
       type: integer
       x_unit: second
+  output_3:
+    displayOrder: 3
+    label: Dream output
+    description: Contains some random data representing a dream
```

Comment:
very good description :-D

In order to be more accurate, perahps you can read [How the brain constructs dreams](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7279884/#:~:text=Deep%20inside%20the%20temporal%20lobe,to%20remember%2C%20imagine%20and%20dream.&text=Our%20most%20vivid%20dreams%20are,a%20richly%20detailed%20hallucinatory%20experience.) and provide the right inputs ;-)

Reply:
We also have neuroscientists in the Team than can help. 😇